Source/Core/Core/Config/SessionSettings.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/Config/SessionSettings.h"
+
+#include "Common/CPUDetect.h"
+#include "Common/CommonTypes.h"
+#include "Common/Config/Config.h"
+
+namespace Config
+{
+const Info<bool> SESSION_USE_FMA{{System::Session, "Core", "UseFMA"}, cpu_info.bFMA};
```

Comment:
`cpu_info` isn't properly initialized at this point, is it?

Reply:
Ah, now that I'm thinking about this a second time, I don't think this is actually safe :(

`cpu_info` does get initialized during dynamic initialization... But so does this, and the order isn't well-defined. That's annoying. I'm not sure what the best way to design this would be then...

Reply:
Would setting NetSettings/DTM field directly from `cpu_info` not work?

Reply:
That does work, though I wonder if it's going to make things messy for us in the future when we try to migrate the movie config handling code to always using the new config system instead of requiring a handwritten line of code for each setting. It would also get rid of the ability to forcibly enable FMA through command-line options in case someone ever wants that – not that that would be a super important feature.

For now, I've made this line create a new temporary instance of `CpuInfo`. Simple and safe, and I don't believe there's any meaningful performance impact.

Reply:
Yeah, I suppose that's a good enough solution for now.